docs/resources/control-flow/user-interactivity/forms/form-widgets/TextField.md

```diff
@@ -0,0 +1,906 @@
+---
+slug: textfield
+title: TextField
+tags: []
+description: Learn how to add TextField widget in your FlutterFlow app.
+sidebar_position: 0
+---
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# TextField
+
+The *TextField* widget allows users to enter text, numbers, and symbols in your app. You can use the *TextField* widget to build forms, send messages, dialogs, search, etc.
+
```

Comment:
Use bold for Textfield instead of italics. the mix of italics and straight font in this page looks odd. check for other references. also we dont have to highlight it in EACH reference, only the first reference in the paragraph should be ok i guess. And any reference to any other widget would probably be a hyperlink. Liek AlertDialog. (Again only the first reference, not every ref)

Reply:
Agree! This was directly from the old docs. So updated accordingly now.